docs/NooBaaNonContainerized/NooBaaCLI.md

```diff
@@ -0,0 +1,603 @@
+
+- `anonymous`
+    - Type: Boolean
+    - Description: Identify the account by the anonymous flag instead of identification by name.
+
+- `show_secrets`
+    - Type: Boolean
+    - Description: Display the access key and secret key of each account (only when using flag --wide).
+
+### List accounts
+
+The `account list` command is used to display a list of all existing accounts.
+
+
+#### Usage
+```sh
+noobaa-cli account list [--wide][--show_secrets][--name][--uid][--gid][--user][--access_key]
+```
+#### Flags -
+- `wide`
+    - Type: Boolean
+    - Description: Display additional details for each bucket.
+
+- `show_secrets`
+    - Type: Boolean
+    - Description: Display the access key and secret key of each account (only when using flag --wide).
+
+- `name`
+    - Type: String
+    - Description: Filter the list based on the provided account name.
+
+- `uid`
+    - Type: String
+    - Description: Filter the list based on the provided account file system uid.
+
+- `gid`
+    - Type: String
+    - Description: Filter the list based on the provided account file system gid.
+
+- `user`
+    - Type: String
+    - Description: Filter the list based on the provided account file system user name.
+
+- `access_key`
+    - Type: String
+    - Description: Filter the list based on the provided account access key.
+
+
+### Delete Account
+
+The `account delete` command is used to delete an existing account.
+
+
+#### Usage
+```sh
+noobaa-cli account delete --name <account_name> [--anonymous]
+```
+#### Flags -
+- `name` (Required)
+    - Type: String
+    - Description: Specifies the name of the bucket to be deleted.
+
+- `anonymous`
+    - Type: Boolean
+    - Description: Identify the account by the anonymous flag instead of identification by name.
+
+## Managing Buckets
+
+- **[Add Bucket](#add-bucket)**: Create new buckets with customizable options.
+- **[Update Bucket](#update-bucket)**: Modify the settings and configurations of existing buckets.
+- **[Bucket Status](#bucket-status)**: Retrieve the current status and detailed information about specific buckets.
+- **[List Buckets](#list-buckets)**: Display a list of all existing buckets, including relevant details.
+- **[Delete Bucket](#delete-bucket)**: Remove unwanted or obsolete buckets from the system.
+
+### Add Bucket
+
+The `bucket add` command is used to create a new bucket with customizable options.
+
+#### Usage
+```sh
+noobaa-cli bucket add --name <bucket_name> --owner <owner_name> --path <path> 
+[--bucket_policy][--fs_backend][--force_md5_etag][--from_file]
+```
+#### Flags -
+- `name` (Required)
+    - Type: String
+    - Description: Specifies the name of the new bucket.
+
+- `owner` (Required)
+    - Type: String
+    - Description: Specifies the name of the owner of the bucket.
+
+- `path` (Required)
+    - Type: String
+    - Description: Specifies the underlying path of the bucket. Path must be an absolute path.
```

Comment:
I think we should answer raising question - is there any connection to the `new_bucket_path` that the account has?
What is the difference between creating a bucket using the CLI and the S3 API?